### PR TITLE
added no data alert for multiple selections of Organisations

### PR DESCRIPTION
**Pull request by HoussamBedja**
To see the changes:
- Select one combination that has 0 data returned.
- Select multiple organizations, including some that have no data (for example, pick All organizations, medium, small and very small).

### CODE REVIEW

File: javascript/load_choice.js [hunk 1]
```diff
@@ -105,11 +106,68 @@ dispatch.on("load_choice", function (load_data, sos_graph_data, question_info) {
...
+                if (current_depts.length > 1){
+                
+                    let available_depts = [];
+
+                    for (i = 0; i < new_TBL_data.length ; i++) {
+                        if (!_.contains(available_depts, new_TBL_data[i].DEPT)) {
+                            available_depts.push(new_TBL_data[i].DEPT);
+                        }
+                    }
+                    let nodata_depts = _.difference(current_depts, available_depts);
```
Comment (dominicdemers) [inline javascript/load_choice.js:137]: good idea to use the diff function!

**Comment by HoussamBedja** [comment]
Just fixed the condition where the "no data" message stays when going from 3 to 2 selections. Should work fine now!

File: javascript/load_table.js [hunk 2]
```diff
@@ -30,7 +32,15 @@ dispatch.on("load_table", function (tbl_data) {
     let table = d3.select("#table_div")
                     .append('table')
                     .attr("id", "adv_tbl")
-                    .attr("class", "table table-condensed");
+                    //.attr("class", "table table-condensed");
+                    .attr("class","wb-tables table table-striped table-hover")
```
Comment (dominicdemers) [inline javascript/load_table.js]: As mentionned, the table sort does not work, we will decline for now, until it is resolved

File: javascript/load_table.js [hunk 3]
```diff
@@ -30,7 +32,15 @@ dispatch.on("load_table", function (tbl_data) {
...
+
+    $(document).ready( function () {
```
Comment (dominicdemers) [inline javascript/load_table.js:37]: Same here, decline until the table sorted is working

File: multi_bar.html [hunk 4]
```diff
@@ -284,11 +284,14 @@ <h1 id="wb-cont" property="name">Staffing and Non-Partisanship Survey interactiv
 	<script src="https://d3js.org/d3.v4.min.js"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore.js" type="text/javascript"></script>
     <link rel="stylesheet" type="text/css" href="css/bar_chart.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.css">
```
Comment (dominicdemers) [inline multi_bar.html:287]: We can leave in, but is not necessary since the table sort does not work

**Review by dominicdemers — COMMENTED** [review]
Please only leave in the code for the warning when there is no data. remove the table sort code until the issues are resolved.

File: javascript/load_table.js (hunk 2 above)
Comment (HoussamBedja) [reply javascript/load_table.js]: I think you commented right when I pushed the fixes. The issue is resolved and this version has the sorting working fine.

**Comment by HoussamBedja** [comment]
The sorting issue has been resolved. The last commit included all the fixes.

**Pull request merged**